src_c/mixer.c

```diff
@@ -790,6 +790,35 @@ snd_get_raw(PyObject *self, PyObject *_null)
                                      (Py_ssize_t)chunk->alen);
 }
 
+static PyObject *
+snd_copy(PyObject *self, PyObject *_null)
+{
+    Mix_Chunk *new_chunk;
+    Mix_Chunk *old_chunk = pgSound_AsChunk(self);
+    pgSoundObject *sound =
+        (pgSoundObject *)Py_TYPE(self)->tp_new(Py_TYPE(self), NULL, NULL);
+
+    if (sound) {
+        new_chunk = Mix_QuickLoad_RAW(old_chunk->abuf, old_chunk->alen);
+        if (new_chunk) {
+            new_chunk->volume = old_chunk->volume;
+        }
+        else {
+            PyErr_Format(PyExc_SystemError,
+                         "Internal pygame error, chunk could not be created.");
```

Comment:
Here, and below we are creating an error, but not immediately exiting the C code via a return, which I think is the normal practice in the pygame codebase.

I assume it won't cause too many problems here, as no doubt these functions return NULL when erroring already, so we'll just end up with returning NULL in the end. However it would probably be a good idea to be explicit that we return NULL/0 here on error by just sticking a return in front of `PyErr_Format`.